docs/ref/contrib/sitemaps.txt

```diff
@@ -188,6 +188,10 @@ Sitemap class reference
         Django respond appropriately to requests with an ``If-Modified-Since``
         header which will prevent sending the sitemap if it hasn't changed.
 
+        .. versionadded:: 1.10
+
+            Default template now includes time value of :attr:`~Sitemap.lastmod`
```

Comment:
"The value of :attr:`~Sitemap.lastmod` was added to the default sitemap template."